hcx-core/hcx-common/src/test/java/org/swasth/common/dto/NotificationTest.java

```diff
@@ -21,43 +22,42 @@ public void testNotificationMockData() {
         Notification mockNotification = Mockito.mock(Notification.class);
         assertNull(mockNotification.getCategory());
         assertNull(mockNotification.getDescription());
-        assertEquals(new ArrayList<>(), mockNotification.getEventType());
         assertNull(mockNotification.getStatus());
-        assertNull(mockNotification.getId());
-        assertNull(mockNotification.getName());
-        assertEquals(new ArrayList<>(), mockNotification.getRecipient());
-        assertEquals(new ArrayList<>(),mockNotification.getSender());
+        assertNull(mockNotification.getTopicCode());
+        assertNull(mockNotification.getTitle());
+        assertEquals(new HashSet<>(), mockNotification.getAllowedRecipients());
+        assertEquals(new HashSet<>(),mockNotification.getAllowedSenders());
         assertNull(mockNotification.getTemplate());
         assertNull(mockNotification.getTrigger());
         assertNull(mockNotification.getType());
     }
 
     @Test
-    public void testNotificationData() throws Exception {
+    public void testNotificationData() {
         Notification notification = new Notification();
         notification.setCategory("Category");
         notification.setDescription("Description");
-        notification.setEventType(null);
-        notification.setId("Id");
-        notification.setName("Name");
-        notification.setRecipient(null);
-        notification.setSender(null);
+        notification.setTopicCode("Id");
+        notification.setTitle("Name");
+        notification.setAllowedRecipients(null);
+        notification.setAllowedSenders(null);
         notification.setTemplate("Template");
         notification.setTrigger("Trigger");
         notification.setType("Type");
         notification.setStatus("Status");
+        notification.setPriority(0);
 
         assertEquals("Category", notification.getCategory());
         assertEquals("Description",notification.getDescription());
-        assertNull(notification.getEventType());
         assertEquals("Status",notification.getStatus());
-        assertEquals("Id",notification.getId());
-        assertEquals("Name",notification.getName());
-        assertNull(notification.getRecipient());
-        assertNull(notification.getSender());
+        assertEquals("Id",notification.getTopicCode());
+        assertEquals("Name",notification.getTitle());
+        assertNull(notification.getAllowedRecipients());
+        assertNull(notification.getAllowedSenders());
```

Comment:
Please add assertion at the data level for all the fields.

Reply:
updated the test cases.